DEFS

```diff
@@ -241,3 +245,14 @@ def standard_java_test(run_test_separately = False, vm_args = None, fork_mode =
           run_test_separately = run_test_separately,
           fork_mode = fork_mode,
         )
+
+def set_error_prone(kwargs):
+  kwargs['javac_jar'] = '//third-party/java/errorprone:error_prone'
+  kwargs['compiler_class_name'] = 'com.google.errorprone.ErrorProneJavaCompiler'
+  ext = 'extra_arguments'
+  if ext not in kwargs:
+    kwargs[ext] = []
+  extra_args = kwargs[ext]
+  # Immutables has an issue with OptionalEquality, that is flagged by error prone
+  extra_args.extend(['-Xep:OptionalEquality:OFF'])
```

Comment:
Oh, I see, that the same issue was raised by @tbroyer on error prone issue tracker, and the solution was to skip warnings in generated code by `-XepDisableWarningsInGeneratedCode`.